Cargo.toml

```diff
@@ -70,6 +69,7 @@ ipnet = { version = "2.11.0", default-features = false, features = [] }
 iptrie = { version = "0.10.3", default-features = false, features = [] }
 mio = { version = "1.0.3", default-features = false, features = [] }
 multi_index_map = { version = "0.12.1", default-features = false, features = [] }
+netdev = { git = "https://github.com/githedgehog/netdev", branch = "pr/daniel-noland/bump-rtnetlink", default-features = false, features = [] }
```

Comment:
The upstream PR you created and mentioned in the commit log has been merged, so this change can probably be reverted?

Reply:
lovely.  Will do